### ISSUE TITLE
android.os.BadParcelableException: ClassNotFoundException when unmarshalling: 

### ISSUE THREAD

**Issue by AppacheCodesandra** [U1F4F1/BetterBottomSheet#4]
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.one20.maps/com.one20.maps.explore.ExploreActivity}: android.os.BadParcelableException: ClassNotFoundException when unmarshalling: 
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2947)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
Caused by android.os.BadParcelableException: ClassNotFoundException when unmarshalling: 
       at android.os.Parcel.readParcelableCreator(Parcel.java:2545)
       at android.os.Parcel.readParcelable(Parcel.java:2471)
       at android.os.Parcel.readParcelableArray(Parcel.java:2579)
       at android.support.design.widget.CoordinatorLayout$SavedState.<init>(CoordinatorLayout.java:3192)
       at android.support.design.widget.CoordinatorLayout$SavedState$1.createFromParcel(CoordinatorLayout.java:3227)
       at android.support.design.widget.CoordinatorLayout$SavedState$1.createFromParcel(CoordinatorLayout.java:3224)
       at android.os.Parcel.readParcelable(Parcel.java:2478)
       at android.os.Parcel.readValue(Parcel.java:2374)
       at android.os.Parcel.readSparseArrayInternal(Parcel.java:2808)
       at android.os.Parcel.readSparseArray(Parcel.java:2077)
       at android.os.Parcel.readValue(Parcel.java:2431)
       at android.os.Parcel.readArrayMapInternal(Parcel.java:2727)
       at android.os.BaseBundle.unparcel(BaseBundle.java:269)
       at android.os.Bundle.getSparseParcelableArray(Bundle.java:910)
       at com.android.internal.policy.PhoneWindow.restoreHierarchyState(PhoneWindow.java:2240)
       at android.app.Activity.onRestoreInstanceState(Activity.java:1126)
       at android.app.Activity.performRestoreInstanceState(Activity.java:1081)
       at android.app.Instrumentation.callActivityOnRestoreInstanceState(Instrumentation.java:1182)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2920)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
```

**Comment by AppacheCodesandra** [comment]
The issue is that the system is trying to unmarshall a class from this library with the default system classloader, so we just need to make sure that we're supplying our own

**Comment by AppacheCodesandra** [comment]
Fixed with release 1.5
